custom_components/smart_irrigation/translations/en.json

```diff
@@ -158,6 +158,50 @@
           "description": "New value of the state"
         }
       }
+    },
+    "set_duration": {
```

Comment:
Oups, I think I left this entry in here by mistake

Reply:
I don't have time to do this, can you please make the change and test?